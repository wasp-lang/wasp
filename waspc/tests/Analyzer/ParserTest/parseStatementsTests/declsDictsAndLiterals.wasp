test Decl {
  string: "Hello Wasp =}",
  escapedString: "Look, a \"",
  integer: 42,
  real: 3.14,
  yes: true,
  no: false,
  ident: Wasp,
  // This is a comment
  innerDict: { innerDictReal: 2.17 }
}
