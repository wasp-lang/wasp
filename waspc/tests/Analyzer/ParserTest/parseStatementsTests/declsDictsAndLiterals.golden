(AST
  (Decl@1:1-11:1 type=test name=Decl
    (Dict@1:11-11:1
      (DictEntry key=string
        (String@2:11-25 value="Hello Wasp =}")
      )
      (DictEntry key=escapedString
        (String@3:18-29 value="Look, a \"")
      )
      (DictEntry key=integer
        (Integer@4:12-13 value=42)
      )
      (DictEntry key=real
        (Double@5:9-12 value=3.14)
      )
      (DictEntry key=yes
        (Bool@6:8-11 value=True)
      )
      (DictEntry key=no
        (Bool@7:7-11 value=False)
      )
      (DictEntry key=ident
        (Var@8:10-13 variable=Wasp)
      )
      (DictEntry key=innerDict
        (Dict@10:14-36
          (DictEntry key=innerDictReal
            (Double@10:31-34 value=2.17)
          )
        )
      )
    )
  )
)
